packages/primitives/radio/src/radio-root.directive.ts

```diff
@@ -14,26 +15,29 @@ import { RadioGroupDirective, RadioGroupProps, RDX_RADIO_GROUP } from './radio-t
     hostDirectives: [{ directive: RdxRovingFocusGroupDirective, inputs: ['dir', 'orientation', 'loop'] }],
     host: {
         role: 'radiogroup',
-        '[attr.aria-orientation]': 'orientation',
-        '[attr.data-disabled]': 'disabled ? "" : null',
+        '[attr.aria-orientation]': 'orientation()',
+        '[attr.data-disabled]': 'disableState() ? "" : null',
         '(keydown)': 'onKeydown()'
     }
 })
 export class RdxRadioGroupDirective implements RadioGroupProps, RadioGroupDirective, ControlValueAccessor {
-    readonly value = model<string | undefined>();
+    readonly value = model<string | null>(null);
 
-    @Input({ transform: booleanAttribute }) disabled = false;
+    readonly disabled = input<boolean, BooleanInput>(false, { transform: booleanAttribute });
 
     @Input() defaultValue?: string;
 
     @Input() required: boolean;
 
-    @Input() orientation: Orientation;
+    readonly orientation = input<Orientation>();
 
     /**
      * Event handler called when the value changes.
      */
-    @Output() readonly onValueChange = new EventEmitter<string>();
+    readonly onValueChange = output<string>();
+
+    private readonly disable = signal<boolean>(this.disabled());
```

Comment:
why you create additional signal, why not use existing `this.disabled`?

Reply:
https://github.com/radix-ng/primitives/pull/186/files/cb04d38348b28aebddcb2bc764c1941a587afd4f#diff-338de32d2976513b4c2ce8e8dbf7ea07cf2cda48ae236981aca972c09ca32293R98

cause need set local state